main.tf

```diff
@@ -46,7 +46,7 @@ resource "aws_security_group" "default" {
 
 # Using tf_instance module
 module "instance" {
```

Comment:
Update module according to the latest [tf_instance](https://github.com/cloudposse/tf_instance/blob/master/README.md) features.